mods/default/tools.lua

```diff
@@ -330,3 +330,70 @@ minetest.register_tool("default:sword_diamond", {
 		damage_groups = {fleshy=8},
 	}
 })
+
+minetest.register_tool("default:skeleton_key", {
+	description = "Skeleton Key",
+	inventory_image = "default_key_skeleton.png",
+	groups = {key = 1},
+	on_place = function(itemstack, placer, pointed_thing)
+		if pointed_thing.type ~= "node" then
+			return nil
+		end
+
+		local pos = pointed_thing.under
+		local node = minetest.get_node(pos)
+
+		if not node or node.name == "ignore" then
```

Comment:
Cool, we don't see enough ignore checks around :smiley:

Btw. `get_node_or_nil`, does return `nil` for it, but `get_node` never does so you probably just need either of those options to check for it here.